src/pudl/metadata/classes.py

```diff
@@ -1863,22 +1863,6 @@ def to_sql(
                 )
         return metadata
 
-    def get_sorted_resources(self) -> StrictList(Resource):
-        """Get a list of sorted Resources.
-
-        Currently Resources are listed in reverse alphabetical order based
-        on their name which results in the following order to promote output
-        tables to users and push intermediate tables to the bottom of the
-        docs: output, core, intermediate.
-
-        In the future we might want to have more fine grain control over how
-        Resources are sorted.
-
-        Returns:
-            A sorted list of resources.
-        """
-        return sorted(self.resources, key=lambda r: r.name, reverse=True)
-
```

Comment:
I added this a while back hoping it would resolve the datasette sorting issue. It does not and I'm going to tackle it in a separate PR.